2013 jun recap

dylancashman opened the pull request:
Added the 2013 jun recap.

Commit:
added jun 2013 recap along with pictures, namespaced sponsors method

Commit:
Merge branch 'master' into 2013_jun_recap

bcarreno commented on web.rb:
```diff
@@ -115,12 +115,12 @@ def load_student
 
   get '/' do
-    @sponsors = sponsors_urls_logos
+    @sponsors = 2013_jun_sponsors_urls_logos
```
Are you sure this is valid ruby?

dylancashman replied:
I'm not sure what you mean.  It seemed to work.  I thought that sponsor method should include the month, since I didn't want it to change the sponsors on my recap page when we get new sponsors.

What's the ruby problem that you see?

danchoi replied:
I think @bcarreno means that Ruby identifiers can't begin with a digit, at least according to some documentation out there.

http://en.wikibooks.org/wiki/Ruby_Programming/Syntax/Lexicology#Identifiers

bcarreno commented:
I read it and it's great writing!, I felt inspired.
Thanks @dylancashman !

danchoi commented:
@dylancashman Thank you for doing this Dylan! Would you like to announce it to the Railsbridge Boston and BostonRB Google Groups when it's live on the web?

dylancashman commented:
I'd be happy to announce it on those mailing lists.  Does anybody else want to do a quick check to see if I didn't screw anything up and then merge the pull request in and deploy it?  I'm not sure how this gets pushed up.

Thanks!

danchoi commented:
Let's aim to push around 1:30pm. 

On Fri, Jul 26, 2013 at 12:28 PM, Dylan Cashman notifications@github.com wrote:

> from: Dylan Cashman notifications@github.com
> date: Fri, Jul 26 09:28 AM -07:00 2013
> to: railsbridge-boston/railsbridge-boston railsbridge-boston@noreply.github.com
> cc: Daniel Choi dhchoi@gmail.com
> reply-to: railsbridge-boston/railsbridge-boston reply@reply.github.com
> subject: Re: [railsbridge-boston] 2013 jun recap (#84)
> 
> I'd be happy to announce it on those mailing lists.  Does anybody else want to do a quick check to see if I didn't screw anything up and then merge the pull request in and deploy it?  I'm not sure how this gets pushed up.
> 
> Thanks!
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/railsbridge-boston/railsbridge-boston/pull/84#issuecomment-21631856

---

Sent from Vmail
http://danielchoi.com/software/vmail.html

bcarreno commented:
A method name can't start with a number.
Besides that I don't see any problem.
Thanks @dylancashman !

bcarreno commented:
@dylancashman if you need any help merging or deploying let me know and I can help you.

danchoi commented:
I think we should also post this and the Video together as a Hacker News
item. 

Let's embed the video in Dylan's recap and post the Hacker News item
before Dylan announces, and in the announcement let's ask people to help
vote up the Hacker News item.

Note that the key to upvoting a Hacker News items is not to link people
directly to the Hacker News posting. Hacker News will discount upvotes
obtained that way. Upvoters should find the HN item indirectly and then
upvote it.

On Fri, Jul 26, 2013 at 12:31 PM, Daniel Choi dhchoi@gmail.com wrote:

> from: Daniel Choi dhchoi@gmail.com
> date: Fri, Jul 26 12:31 PM -04:00 2013
> to: railsbridge-boston/railsbridge-boston reply@reply.github.com
> cc: railsbridge-boston/railsbridge-boston railsbridge-boston@noreply.github.com, dhchoi@gmail.com
> subject: Re: [railsbridge-boston] 2013 jun recap (#84)
> 
> Let's aim to push around 1:30pm. 
> 
> On Fri, Jul 26, 2013 at 12:28 PM, Dylan Cashman notifications@github.com wrote:
> 
> > from: Dylan Cashman notifications@github.com
> > date: Fri, Jul 26 09:28 AM -07:00 2013
> > to: railsbridge-boston/railsbridge-boston railsbridge-boston@noreply.github.com
> > cc: Daniel Choi dhchoi@gmail.com
> > reply-to: railsbridge-boston/railsbridge-boston reply@reply.github.com
> > subject: Re: [railsbridge-boston] 2013 jun recap (#84)
> > 
> > I'd be happy to announce it on those mailing lists.  Does anybody else want to do a quick check to see if I didn't screw anything up and then merge the pull request in and deploy it?  I'm not sure how this gets pushed up.
> > 
> > Thanks!
> > 
> > ---
> > 
> > Reply to this email directly or view it on GitHub:
> > https://github.com/railsbridge-boston/railsbridge-boston/pull/84#issuecomment-21631856
> 
> ---
> 
> Sent from Vmail
> http://danielchoi.com/software/vmail.html

---

Sent from Vmail
http://danielchoi.com/software/vmail.html

Commit:
changing invalid method name and adding video teaser to recap

dylancashman commented:
Ok, I changed the method name and I added in the video teaser into the recap.  I don't know how to deploy, actually, so @bcarreno if you wanted to do that, that would be great.

danchoi commented:
@bcarreno @dylancashman I got this. I'll merge and deploy

drapergeek commented on views/blog/2013_jun_recap_text.markdown:
```diff
@@ -0,0 +1,229 @@
+
+### Childcare
+
+Thanks to sponsorship from [Thoughtbot](http://www.thoughtbot.com/), we 
```
thoughtbot should not be capitalized.

drapergeek commented on views/blog/2013_jun_recap_text.markdown:
```diff
@@ -0,0 +1,229 @@
+>  made childcare possible. I felt entirely comfortable leaving my 
+>  two-year-old with the person the service sent, and without the 
+>  childcare I wouldn't have been able to attend at all. I appreciate 
+>  the organizers thinking of (single) moms who want to code more than 
```
Did the person who wrote this specify single? If not I'm not sure we should put that in.

dylancashman replied:
It was a direct quote from their survey.

danchoi replied:
OK I fixed the capitalization of thoughtbot, the latest version is actually
live.

http://www.railsbridgeboston.org/blog/2013_jun_recap

I'll post a Hacker News item. Dylan, can you come up with a Hacker News
item title?

On Fri, Jul 26, 2013 at 12:57 PM, Dylan Cashman notifications@github.comwrote:

> In views/blog/2013_jun_recap_text.markdown:
> 
> > +lauded feature of the workshop amongst students, and we feel like
> > +it is the key to our success.
> > +
> > +### Childcare
> > +
> > +Thanks to sponsorship from [Thoughtbot](http://www.thoughtbot.com/), we
> > +were able to ofter childcare assistance during the workshop.  One
> > +mother left us particularly kind feedback.
> > +
> > +---
> > +
> > +>  I'd like to extend the most heartfelt of thank-yous to those who
> > +>  made childcare possible. I felt entirely comfortable leaving my
> > +>  two-year-old with the person the service sent, and without the
> > +>  childcare I wouldn't have been able to attend at all. I appreciate
> > +>  the organizers thinking of (single) moms who want to code more than
> 
> It was a direct quote from their survey.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/railsbridge-boston/railsbridge-boston/pull/84/files#r5427249
> .

dylancashman replied:
How about "Rails Workshop for Women in Boston - Recap"

danchoi replied:
I think we need some that grabs attention

e.g.

"RailsBridge Boston welcome 64 women into the Boston Ruby community, and
produces a video"

On Fri, Jul 26, 2013 at 1:09 PM, Dylan Cashman notifications@github.comwrote:

> In views/blog/2013_jun_recap_text.markdown:
> 
> > +lauded feature of the workshop amongst students, and we feel like
> > +it is the key to our success.
> > +
> > +### Childcare
> > +
> > +Thanks to sponsorship from [Thoughtbot](http://www.thoughtbot.com/), we
> > +were able to ofter childcare assistance during the workshop.  One
> > +mother left us particularly kind feedback.
> > +
> > +---
> > +
> > +>  I'd like to extend the most heartfelt of thank-yous to those who
> > +>  made childcare possible. I felt entirely comfortable leaving my
> > +>  two-year-old with the person the service sent, and without the
> > +>  childcare I wouldn't have been able to attend at all. I appreciate
> > +>  the organizers thinking of (single) moms who want to code more than
> 
> How about "Rails Workshop for Women in Boston - Recap"
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/railsbridge-boston/railsbridge-boston/pull/84/files#r5427456
> .

danchoi replied:
Jason I pushed all your corrections

On Fri, Jul 26, 2013 at 1:12 PM, Daniel Choi dhchoi@gmail.com wrote:

> I think we need some that grabs attention
> 
> e.g.
> 
> "RailsBridge Boston welcome 64 women into the Boston Ruby community, and
> produces a video"
> 
> On Fri, Jul 26, 2013 at 1:09 PM, Dylan Cashman notifications@github.comwrote:
> 
> > In views/blog/2013_jun_recap_text.markdown:
> > 
> > > +lauded feature of the workshop amongst students, and we feel like
> > > +it is the key to our success.
> > > +
> > > +### Childcare
> > > +
> > > +Thanks to sponsorship from [Thoughtbot](http://www.thoughtbot.com/), we
> > > +were able to ofter childcare assistance during the workshop.  One
> > > +mother left us particularly kind feedback.
> > > +
> > > +---
> > > +
> > > +>  I'd like to extend the most heartfelt of thank-yous to those who
> > > +>  made childcare possible. I felt entirely comfortable leaving my
> > > +>  two-year-old with the person the service sent, and without the
> > > +>  childcare I wouldn't have been able to attend at all. I appreciate
> > > +>  the organizers thinking of (single) moms who want to code more than
> > 
> > How about "Rails Workshop for Women in Boston - Recap"
> > 
> > —
> > Reply to this email directly or view it on GitHubhttps://github.com/railsbridge-boston/railsbridge-boston/pull/84/files#r5427456
> > .

drapergeek replied:
:cool:

danchoi replied:
OK we are live and there is a Hacker News item! We can still make
corrections after the fact.

Dylan, can you post to the Google Groups and also ask for upvotes? Remember
not to link them directly to the hacker new post, but just give them the
title of the item and ask them to find it on

https://news.ycombinator.com/newest

RailsBridge Boston welcomes 64 women into Boston Ruby community (w/
video)http://www.railsbridgeboston.org/blog/2013_jun_recap

On Fri, Jul 26, 2013 at 1:14 PM, Jason Draper notifications@github.comwrote:

> In views/blog/2013_jun_recap_text.markdown:
> 
> > +lauded feature of the workshop amongst students, and we feel like
> > +it is the key to our success.
> > +
> > +### Childcare
> > +
> > +Thanks to sponsorship from [Thoughtbot](http://www.thoughtbot.com/), we
> > +were able to ofter childcare assistance during the workshop.  One
> > +mother left us particularly kind feedback.
> > +
> > +---
> > +
> > +>  I'd like to extend the most heartfelt of thank-yous to those who
> > +>  made childcare possible. I felt entirely comfortable leaving my
> > +>  two-year-old with the person the service sent, and without the
> > +>  childcare I wouldn't have been able to attend at all. I appreciate
> > +>  the organizers thinking of (single) moms who want to code more than
> 
> [image: :cool:]
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/railsbridge-boston/railsbridge-boston/pull/84/files#r5427547
> .

danchoi replied:
Thank you Dylan!

On Fri, Jul 26, 2013 at 1:18 PM, Daniel Choi dhchoi@gmail.com wrote:

> OK we are live and there is a Hacker News item! We can still make
> corrections after the fact.
> 
> Dylan, can you post to the Google Groups and also ask for upvotes?
> Remember not to link them directly to the hacker new post, but just give
> them the title of the item and ask them to find it on
> 
> https://news.ycombinator.com/newest
> 
> RailsBridge Boston welcomes 64 women into Boston Ruby community (w/ video)http://www.railsbridgeboston.org/blog/2013_jun_recap
> 
> On Fri, Jul 26, 2013 at 1:14 PM, Jason Draper notifications@github.comwrote:
> 
> > In views/blog/2013_jun_recap_text.markdown:
> > 
> > > +lauded feature of the workshop amongst students, and we feel like
> > > +it is the key to our success.
> > > +
> > > +### Childcare
> > > +
> > > +Thanks to sponsorship from [Thoughtbot](http://www.thoughtbot.com/), we
> > > +were able to ofter childcare assistance during the workshop.  One
> > > +mother left us particularly kind feedback.
> > > +
> > > +---
> > > +
> > > +>  I'd like to extend the most heartfelt of thank-yous to those who
> > > +>  made childcare possible. I felt entirely comfortable leaving my
> > > +>  two-year-old with the person the service sent, and without the
> > > +>  childcare I wouldn't have been able to attend at all. I appreciate
> > > +>  the organizers thinking of (single) moms who want to code more than
> > 
> > [image: :cool:]
> > 
> > —
> > Reply to this email directly or view it on GitHubhttps://github.com/railsbridge-boston/railsbridge-boston/pull/84/files#r5427547
> > .

danchoi replied:
We made the bottom of the front page!

https://news.ycombinator.com/

On Fri, Jul 26, 2013 at 1:43 PM, Daniel Choi dhchoi@gmail.com wrote:

> Thank you Dylan!
> 
> On Fri, Jul 26, 2013 at 1:18 PM, Daniel Choi dhchoi@gmail.com wrote:
> 
> > OK we are live and there is a Hacker News item! We can still make
> > corrections after the fact.
> > 
> > Dylan, can you post to the Google Groups and also ask for upvotes?
> > Remember not to link them directly to the hacker new post, but just give
> > them the title of the item and ask them to find it on
> > 
> > https://news.ycombinator.com/newest
> > 
> > RailsBridge Boston welcomes 64 women into Boston Ruby community (w/ video)http://www.railsbridgeboston.org/blog/2013_jun_recap
> > 
> > On Fri, Jul 26, 2013 at 1:14 PM, Jason Draper notifications@github.comwrote:
> > 
> > > In views/blog/2013_jun_recap_text.markdown:
> > > 
> > > > +lauded feature of the workshop amongst students, and we feel like
> > > > +it is the key to our success.
> > > > +
> > > > +### Childcare
> > > > +
> > > > +Thanks to sponsorship from [Thoughtbot](http://www.thoughtbot.com/), we
> > > > +were able to ofter childcare assistance during the workshop.  One
> > > > +mother left us particularly kind feedback.
> > > > +
> > > > +---
> > > > +
> > > > +>  I'd like to extend the most heartfelt of thank-yous to those who
> > > > +>  made childcare possible. I felt entirely comfortable leaving my
> > > > +>  two-year-old with the person the service sent, and without the
> > > > +>  childcare I wouldn't have been able to attend at all. I appreciate
> > > > +>  the organizers thinking of (single) moms who want to code more than
> > > 
> > > [image: :cool:]
> > > 
> > > —
> > > Reply to this email directly or view it on GitHubhttps://github.com/railsbridge-boston/railsbridge-boston/pull/84/files#r5427547
> > > .

drapergeek replied:
Can someone who controls the twitter account toss the link out for upvotes?

danchoi replied:
I'll do it!

On Fri, Jul 26, 2013 at 2:00 PM, Jason Draper notifications@github.comwrote:

> In views/blog/2013_jun_recap_text.markdown:
> 
> > +lauded feature of the workshop amongst students, and we feel like
> > +it is the key to our success.
> > +
> > +### Childcare
> > +
> > +Thanks to sponsorship from [Thoughtbot](http://www.thoughtbot.com/), we
> > +were able to ofter childcare assistance during the workshop.  One
> > +mother left us particularly kind feedback.
> > +
> > +---
> > +
> > +>  I'd like to extend the most heartfelt of thank-yous to those who
> > +>  made childcare possible. I felt entirely comfortable leaving my
> > +>  two-year-old with the person the service sent, and without the
> > +>  childcare I wouldn't have been able to attend at all. I appreciate
> > +>  the organizers thinking of (single) moms who want to code more than
> 
> Can someone who controls the twitter account toss the link out for upvotes?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/railsbridge-boston/railsbridge-boston/pull/84/files#r5428702
> .

danchoi replied:
OK we're off the front page. It was a good run.

On Fri, Jul 26, 2013 at 2:02 PM, Daniel Choi dhchoi@gmail.com wrote:

> I'll do it!
> 
> On Fri, Jul 26, 2013 at 2:00 PM, Jason Draper notifications@github.comwrote:
> 
> > In views/blog/2013_jun_recap_text.markdown:
> > 
> > > +lauded feature of the workshop amongst students, and we feel like
> > > +it is the key to our success.
> > > +
> > > +### Childcare
> > > +
> > > +Thanks to sponsorship from [Thoughtbot](http://www.thoughtbot.com/), we
> > > +were able to ofter childcare assistance during the workshop.  One
> > > +mother left us particularly kind feedback.
> > > +
> > > +---
> > > +
> > > +>  I'd like to extend the most heartfelt of thank-yous to those who
> > > +>  made childcare possible. I felt entirely comfortable leaving my
> > > +>  two-year-old with the person the service sent, and without the
> > > +>  childcare I wouldn't have been able to attend at all. I appreciate
> > > +>  the organizers thinking of (single) moms who want to code more than
> > 
> > Can someone who controls the twitter account toss the link out for
> > upvotes?
> > 
> > —
> > Reply to this email directly or view it on GitHubhttps://github.com/railsbridge-boston/railsbridge-boston/pull/84/files#r5428702
> > .

danchoi replied:
The Hacker News run genereted a good number of tweets

http://topsy.com/www.railsbridgeboston.org/blog/2013_jun_recap

Mission accomplished.

On Fri, Jul 26, 2013 at 2:38 PM, Daniel Choi dhchoi@gmail.com wrote:

> OK we're off the front page. It was a good run.
> 
> On Fri, Jul 26, 2013 at 2:02 PM, Daniel Choi dhchoi@gmail.com wrote:
> 
> > I'll do it!
> > 
> > On Fri, Jul 26, 2013 at 2:00 PM, Jason Draper notifications@github.comwrote:
> > 
> > > In views/blog/2013_jun_recap_text.markdown:
> > > 
> > > > +lauded feature of the workshop amongst students, and we feel like
> > > > +it is the key to our success.
> > > > +
> > > > +### Childcare
> > > > +
> > > > +Thanks to sponsorship from [Thoughtbot](http://www.thoughtbot.com/), we
> > > > +were able to ofter childcare assistance during the workshop.  One
> > > > +mother left us particularly kind feedback.
> > > > +
> > > > +---
> > > > +
> > > > +>  I'd like to extend the most heartfelt of thank-yous to those who
> > > > +>  made childcare possible. I felt entirely comfortable leaving my
> > > > +>  two-year-old with the person the service sent, and without the
> > > > +>  childcare I wouldn't have been able to attend at all. I appreciate
> > > > +>  the organizers thinking of (single) moms who want to code more than
> > > 
> > > Can someone who controls the twitter account toss the link out for
> > > upvotes?
> > > 
> > > —
> > > Reply to this email directly or view it on GitHubhttps://github.com/railsbridge-boston/railsbridge-boston/pull/84/files#r5428702
> > > .

drapergeek commented on views/blog/2013_jun_recap_text.markdown:
```diff
@@ -0,0 +1,229 @@
+graduates several freebies and deals.  Every student was able to 
+choose an [O'Reilly E-book](http://oreilly.com/) for free.  Ryan 
+Bates was kind enough to offer some free subscriptions to 
+[Railscasts](railscasts.com) and [Thoughtbot](http://www.thoughtbot.com) provided 
```
thoughtbot should not be capitalized.

drapergeek commented on views/blog/2013_jun_recap_text.markdown:
```diff
@@ -0,0 +1,229 @@
+a discount to their education service, [Prime](https://learn.thoughtbot.com/prime).  
+
+Finally, we encouraged all students to keep in touch with the 
+Railsbridge mailing list.  
```
Railsbridge -> RailsBridge

drapergeek commented on views/blog/2013_jun_recap_text.markdown:
```diff
@@ -0,0 +1,229 @@
+* [ActBlue](http://actblue.com)
+* [Railscasts](http://railscasts.com)
+* [O'Reilly](http://oreilly.com)
+* [Thoughtbot](http://thoughtbot.com)
```
thoughtbot should not be capitalized

drapergeek commented on views/blog/2013_jun_recap_text.markdown:
```diff
@@ -0,0 +1,229 @@
+
+## The next workshop
+
+We plan on holding our next RailsBridge Boston workshop sometime in September.  Watch the [main page](http://www.railsbridgeboston.org/) and follow us on [twitter](https://twitter.com/RailsBridgeBos) to find out more details.
```
The next workshop is in October.
